### PR TITLE
For issue: #16394 Fix UI for consider increasing some elements from ETP-Exceptions page

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/layout/exception_item.xml [hunk 1]
```diff
@@ -48,8 +48,8 @@
 
     <ImageButton
         android:id="@+id/delete_exception"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="48dp"
```
Comment (jonalmeida) [inline app/src/main/res/layout/exception_item.xml:51]: Was this meant to be `component_exceptions_learn_more_height` as well?

**Review by jonalmeida — APPROVED** [review]

File: app/src/main/res/layout/exception_item.xml (hunk 1 above)
Comment (Amejia481) [reply app/src/main/res/layout/exception_item.xml:51]: I think this is specific to the exception item icon, we could add a `dimen` for it

Comment (Amejia481) [reply app/src/main/res/layout/exception_item.xml]: I added `component_tp_exceptions_icon_size` for it.

Comment (jonalmeida) [reply app/src/main/res/layout/exception_item.xml]: Ah sorry, I misread. The dimen was super necessary, but it's useful. :)

Comment (Amejia481) [reply app/src/main/res/layout/exception_item.xml]: 👍🏽

**Pull request merged**